src/index.js

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import MovieRoutes from './routes/MovieRoutes';
+
+
+// Creates a new instance of express
+const app = express();
+
+// Connect to the Mongo database
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/movie-list');
+
+
+// Body-parser is middleware that parses incoming request bodies, and
+// makes the payload available to use through request.body
+app.use(bodyParser.json());
+
+
+// Add MovieRoutes to the application
+app.use(MovieRoutes);
+app.use( (err, request, response) => {
```

Comment:
This error middleware won't work properly, need to supply 4 arguments, not just 3

```
(err, req, res, next)
```

Need to turn off the eslint rule for this block